app/_hub/kong-inc/mocking/index.md

```diff
@@ -0,0 +1,897 @@
+    "updated_at": 1619187402,
+    "write_timeout": 60000
+}
+```
+
+### Step 3. Create the get stock quote route {#create-stock-quote-route}
+
+This example creates a route named `getStockQuote` on the service named `Stock-Service`.
+
+{% navtabs %}
+{% navtab cURL %}
+
+```bash
+curl -X POST http://localhost:8001/services/Stock-Service/routes \
+    --data "name=getStockQuote" \
+    --data paths="/stock/historical"
+```
+
+{% endnavtab %}
+{% navtab HTTPie %}
+
+```bash
+http -f :8001/services/Stock-Service/routes name='getStockQuote' paths="/stock/historical"
+```
+
+{% endnavtab %}
+{% endnavtabs %}
+
+Response:
+
+```
+HTTP/1.1 201 Created
+Access-Control-Allow-Credentials: true
+Access-Control-Allow-Origin: http://localhost:8002
+Connection: keep-alive
+Content-Length: 497
+Content-Type: application/json; charset=utf-8
+Date: Fri, 23 Apr 2021 14:26:44 GMT
+Server: kong/2.4.0.0-beta1-enterprise-edition
+X-Kong-Admin-Latency: 29
+X-Kong-Admin-Request-ID: UtF0twm9PoeI3gD7SqH6XDvm3JRqKm8o
+vary: Origin
+
+{
+    "created_at": 1619188004,
+    "destinations": null,
+    "headers": null,
+    "hosts": null,
+    "https_redirect_status_code": 426,
+    "id": "5ada3c98-c732-4c05-903f-9ba8ef026776",
+    "methods": null,
+    "name": "getStockQuote",
+    "path_handling": "v0",
+    "paths": [
+        "/stock/historical"
+    ],
+    "preserve_host": false,
+    "protocols": [
+        "http",
+        "https"
+    ],
+    "regex_priority": 0,
+    "request_buffering": true,
+    "response_buffering": true,
+    "service": {
+        "id": "94d95e04-bb28-4707-bef7-32307091b5dc"
+    },
+    "snis": null,
+    "sources": null,
+    "strip_path": true,
+    "tags": null,
+    "updated_at": 1619188004
+}
+```
+
+### Step 4. Enable the Mocking plugin {#enable-mock-plugin}
+
+This example enables the Mocking plugin on the `getStockQuote` route.
+
+Command:
+
+{% navtabs %}
+{% navtab cURL %}
+
+```bash
+curl -X POST http://<admin-hostname>:8001/routes/getStockQuote/plugins \
+    --data "name=mocking"  \
+    --data "config.api_specification_filename=stock-0.1.json"
+```
+
+Optional configuration for random simulated delay:
+
+```bash
+curl -X POST http://<admin-hostname>:8001/routes/getStockQuote/plugins \
+    --data "name=mocking"  \
+    --data "config.api_specification_filename=stock-0.1.json" \
+    --data "config.random_delay=true" \
+    --data "config.max_delay_time=1" \
+    --data "config.min_delay_time=0.001"
+```
+
+DB-less or hybrid mode configuration must use `config.api_specification`:
+
+```bash
+curl -X POST http://<admin-hostname>:8001/routes/getStockQuote/plugins \
+    --data "name=mocking"  \
+    --data "config.api_specification=<spec_contents>"
+```
+
+Because a spec can be rather lengthy to put into a command, use a local variable to post
+a spec:
+
+```bash
+mock_ex=$(cat example.yaml); curl -X POST http://<admin-hostname>:8001/routes/<route_id>/plugins --data name=mocking --data config.api_specification="$mock_ex"
```

Comment:
```suggestion
mock_ex=$(cat example.yaml); \
curl -X POST http://<admin-hostname>:8001/routes/<route_id>/plugins \
  --data name=mocking \
  --data config.api_specification="$mock_ex"
```
Split it up to avoid having a long scrollable codeblock